app/src/main/java/com/atom/android/muvik/ui/home/HomeFragment.kt

```diff
@@ -0,0 +1,77 @@
+package com.atom.android.muvik.ui.home
+
+import android.Manifest
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Toast
+import androidx.activity.result.contract.ActivityResultContracts
+import com.atom.android.muvik.R
+import com.atom.android.muvik.base.BaseFragment
+import com.atom.android.muvik.data.model.Song
+import com.atom.android.muvik.data.repository.SongRepository
+import com.atom.android.muvik.data.source.local.SongLocalDataSource
+import com.atom.android.muvik.databinding.FragmentHomeBinding
+import com.atom.android.muvik.ui.main.MusicService
+import com.atom.android.muvik.utils.Constant
+
+
+class HomeFragment : BaseFragment<FragmentHomeBinding>(FragmentHomeBinding::inflate),
+    HomeContract.View, AdapterSongs.ItemClickListener {
+
+    private var adapterSongs = AdapterSongs(this)
```

Comment:
ko khai báo var lung lung 
```suggestion
    private val adapterSongs = AdapterSongs(this)
```